frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -122,7 +124,7 @@ const PVCTableRow = connect(mapStateToProps)(({ obj, index, key, style, metrics
         <ResourceLink kind="Namespace" name={namespace} title={namespace} />
       </TableData>
       <TableData className={tableColumnClasses[2]}>
-        <PVCStatus pvc={obj} />
+        <PVCStatus {...obj} />
```

Comment:
Again, I prefer passing the PVC object as a prop. It's unclear why this was needed.

Reply:
> Again, I prefer passing the PVC object as a prop. It's unclear why this was needed.

Will address.